src/execution_plan/execution_plan.c

```diff
@@ -419,7 +442,7 @@ static inline void _buildProjectionOps(ExecutionPlan *plan, AR_ExpNode **project
 		bool sorting_after_aggregation = (order_exps != NULL);
 		op = NewAggregateOp(plan, projections, sorting_after_aggregation);
```

Comment:
Aggregate ops will need a similar update, though I think that aggregation queries that trigger this class of issue are less common. I can implement once we agree on whether the approach in OpProject is proper!